cms/jinja2/templates/pages/theme_page.html

```diff
@@ -0,0 +1,21 @@
+{% extends "templates/base_page.html" %}
+
+{% block header_area %}
+    <div class="theme-header">
+        <div class="ons-container">
+            {% include "templates/components/navigation/breadcrumbs.html" %}
```

Comment:
I have raised a separate card to deal with breadcrumbs off the back of the decisions in the Nov proto.